.github/workflows/update-contributors.yaml

```diff
@@ -0,0 +1,27 @@
+name: Update Contributors
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'main'
+
+env:
+  UPDATE_CONTRIBUTORS_BRANCH: update-contributors
+
+jobs:
+  add-contributors:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Delete old branch (If it exists)
+        run: git push origin --delete ${{ env.UPDATE_CONTRIBUTORS_BRANCH }}
+        continue-on-error: true
+      - name: Create and push new branch
+        run: |
+          git branch ${{ env.UPDATE_CONTRIBUTORS_BRANCH }}
+          git push origin ${{ env.UPDATE_CONTRIBUTORS_BRANCH }}
+      - uses: BobAnkh/add-contributors@v0.2.2
```

Comment:
![imo-badge](https://img.shields.io/badge/review-imo-orange.svg) 
I think it would be more considerate to comment the URL of the action.
```suggestion
      # https://github.com/BobAnkh/add-contributors
      - uses: BobAnkh/add-contributors@v0.2.2
```